plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -30,7 +30,7 @@ export const mapProcessInstanceToDetails = (
   }
 
   const started = start?.toDate().toLocaleString() ?? VALUE_UNAVAILABLE;
-  const variables = parseWorkflowVariables(instance?.variables);
+  const variables = parseWorkflowVariables(instance?.workflowdata);
   const nextWorkflowSuggestions: WorkflowRunDetail['nextWorkflowSuggestions'] =
```

Comment:
I think we can remove both lines 33 and 34. 
Can you please double check they are not used anywhere else? Git-grep suggests that they are not.

Reply:
the var 'variables' is used by line 34 and that is used in the return value

Reply:
Right. But I do not see the returned value to be used any further.

Reply:
in line 36 we use variables for setting nextWorkflowSuggestions. then we use nextWorkflowSuggestions in returned value

Reply:
and then nextWorkflowSuggestions isn't used
It needs to be removed from the return value
and from the type definition WorkflowRunDetail

Reply:
I think I fixed it. please check